app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -103,10 +104,10 @@ export const promptVersionToInstance = (
  * @param instance - The playground instance
  * @returns The prompt version
  */
-export const instanceToPromptVersion = (instance: PlaygroundInstance) => {
-  if (!instance.prompt) {
-    return null;
-  }
+export const instanceToPromptVersion = (
+  instance: PlaygroundInstance,
+  templateLanguage: TemplateLanguage
```

Comment:
I guess this causes some inconsistency. Ideally the instance self contains the template language?

Reply:
It is currently global across the playground so we'd have to change that as well if we want to change this

Reply:
yeah, it's just a slightly strange signature now. I get it though

Reply:
I just realized that I don't pass description in to this function, I spread it into the result later on. I think I could possibly do the same for template language